lib/canvas.js

```diff
@@ -177,6 +177,14 @@ class NodeCanvasElement extends Canvas {
 
   setAttribute(key, value) {
     this.__attributes__[key] = value;
+
+    if (key === 'width') {
```

Comment:
Should `getAttribute` also be updated? or perhaps the setter for `width` and `height`? Otherwise it's possible to get an outdated value from `getAttribute` if the width or height has changed through their setters.